packages/contract-wrappers/src/types.ts

```diff
@@ -225,3 +225,61 @@ export interface DutchAuctionData {
     beginTimeSeconds: BigNumber;
     beginAmount: BigNumber;
 }
+
+export interface CoordinatorServerApprovalResponse {
+    signatures: string[];
+    expirationTimeSeconds: BigNumber[];
+}
+export interface CoordinatorServerApprovalRawResponse {
+    signatures: string[];
+    expirationTimeSeconds: BigNumber;
+}
+
+export interface CoordinatorServerCancellationResponse {
+    outstandingFillSignatures: [
+        {
+            orderHash: string;
+            approvalSignatures: string[];
+            expirationTimeSeconds: BigNumber;
+            takerAssetFillAmount: BigNumber;
+        }
```

Comment:
Make this a separate type declaration. Our docs cannot handle infinitely nested type definitions yet